### DISCUSSION TITLE
GPS

### DISCUSSION THREAD

**Discussion by npcwoody** [jlai/fitness-dashboard#14] [Ideas]
I used the FitBit Web Dashboard to view my walking rout using GPS. That was py primary use of the dashboard. I could do a screen dump and email it to friends and relatives to show off. Any chance you will look into it in the future?

**Comment by jlai** [comment]
Have you looked in the History -> Activities page? For tracked activities there should be a compass icon next to it.

**Reply by npcwoody** [reply]
Sorry, this is my first day, and I hadn't explored that far yet. Now that I know where to look I should be okay.

**Reply by jlai** [reply]
No worries!

The Fitbit mobile app also has a share GPS map feature, but it's hidden under You -> Community -> Feed (scroll a bit to the right under Community), tap "What are you up to?" -> Exercise -> browse to the activity you want to share a picture of. Talk about exploring :)

**Reply by npcwoody** [reply]
Thank you. Figures it would be hidden.  Will your Dashboard work when I am forced to switch to Google?

**Reply by jlai** [reply]
Yes, I'm using a Fitbit account that was converted to Google login.

**Reply by npcwoody** [reply]
Thanks.
